src/lib/bootloader/exceptions.rb

```diff
@@ -25,4 +25,12 @@ def initialize(msg)
         "Please use YaST2 bootloader to fix it. Details: %s") % msg
     end
   end
+
+  # Represents error when serial console arguments are not valid
+  class InvalidSerialConsoleArguments < RuntimeError
```

Comment:
Why make a new exception class when you only add code to raise `BrokenConfiguration`?